templates/default/customer/customeradd.html

```diff
@@ -122,17 +122,32 @@ <H1>{$layout.pagetitle}</H1>
 			<div class="lmsbox-panel">
 				<table>
 					<tr>
+						<td>
+							<label for="customeradd[extid]" class="nobr bold">
+								{trans("External system identifier")}
+							</label>
+						</td>
 						<td>
 							<div class="valign-middle">
-								<label>
-									<strong>{trans("External system identifier")}</strong>
-									<input type="text" size="20" maxlength="20" name="customeradd[extid]" value="{$customeradd.extid|escape}"
-										{tip trigger="extid"}>
-								</label>
+								<input type="text" class="lms-ui-autogrow" name="customeradd[extid]" value="{$customeradd.extid|escape}"
+									{tip trigger="extid"}>
 								{hint content="Enter customer identifier in external system (optional)"}
```

Comment:
Przemycasz inne zmiany (być może słuszne) z inną zmianą - zasadniczą.

Reply:
Inaczej rozjeżdża się to wizualnie.